source/server/backtrace.h

```diff
@@ -104,7 +109,13 @@ class BackwardsTrace : Logger::Loggable<Logger::Id::backtrace> {
       ENVOY_LOG(critical, "thr<{}> #{} {}: {}", thread_id, stack_trace_[i].idx,
```

Comment:
Nit: we're emitting `"thr<{}> #{} {}: {}"` on macOS and `""thr<{}> #{} {} {}"` (without `:`) on Linux.

Consider removing `:` from macOS output or adding it to both Linux outputs.